app/views/v1/subdomain_util.py

```diff
@@ -0,0 +1,109 @@
+from typing import Set, Tuple
+
+import json
+
+import flask_jwt_extended
+from flask import request, url_for
+
+import app.utils.authentication_utils as authentication_utils
+import app.actions as actions
+import app.db_models as db_models
+
+from . import bp
+from app.utils.ct_search import get_subdomains_from_ct
+from app.actions.add_targets import add_targets
+
+from time import time
+
+BATCH_SIZE = 10  # set appropriate amount
+RESCAN_INTERVAL = 62400  # to check target once a day
+
+
+@bp.route("/rescan_subdomains", methods=["GET"])  # is ok?
+@flask_jwt_extended.jwt_required
+def api_cron_rescan_subdomains() -> None:
+    rescan_subdomains()
+
+
+def rescan_subdomains() -> int:
+    targets_to_rescan = (
+        db_models.db.session.query(db_models.SubdomainRescanTarget)
+        .order_by(db_models.SubdomainRescanTarget.subdomain_last_scan.asc())
+        .limit(BATCH_SIZE).all()
+    )
+    for target in targets_to_rescan:
+        if time() - target.subdomain_last_scan < RESCAN_INTERVAL:
+            continue
+        add_subdomains(target.subdomain_scan_target_id, target.subdomain_scan_user_id)
+        target.subdomain_last_scan = int(time())  # should work, check
+        db_models.db.session.commit()
+    return len(targets_to_rescan)  # for testing
+
+
+def add_subdomains(target_id: int, user_id: int, data) -> Tuple[str, int, int]:
+    target = actions.get_target_from_id_if_user_can_see(target_id, user_id)
+
+    if target is None:
+        return "You do not have permission to track this target.", 0, 400
+
+    if data is None:  # dummy data for searching and enqueuing new subdomains
+        data = {
+                "scanOrder": {"active": None, "periodicity": 43200},
+                "target": {
+                    "hostname": f"{target.hostname}",
+                    "id": None,
+                    "ip_address": None,
+                    "port": None,
+                    "protocol": "HTTPS",
+                },
+            }
```

Comment:
Maybe create tmp `Target` and `ScanOrder` objects and then dump them to JSON via schema. That's more maintainable.